code/index.html

```diff
@@ -1,30 +1,166 @@
+        <input
+          type="password"
+          id="password"
+          name="password"
+          class="input-text-block"
+          placeholder="Password"
+          required
+        />
+
+        <h4 class="form-headline uppercase">What describes you?</h4>
+        <input
+          type="radio"
+          id="teacher"
+          name="teacherOrStudent"
+          class="pointer"
+        />
+        <label class="inline-label pointer" for="teacher">Teacher</label>
+
+        <input
+          type="radio"
+          id="student"
+          name="teacherOrStudent"
+          class="pointer"
+        />
+        <label class="inline-label pointer" for="student">Student</label>
+
+        <h4 class="form-headline uppercase">Terms and conditions</h4>
+        <input
+          type="checkbox"
+          id="termsCond"
+          name="termsCond"
+          class="pointer"
+          required
+        />
+        <label class="inline-label pointer" for="termsCond">I accept the Terms & Conditions</label>
+
+        <br>
+
+        <input
+          type="checkbox"
+          id="newsletter"
+          name="newsletter"
+          class="pointer"
+        />
+        <label class="inline-label pointer" for="newsletter">I want to receive newsletter</label>
+
+        <br>
+
+        <button class="sign-up-btn pointer uppercase" type="submit">
+          Sign up
+        </button>
+
+      </form>
+
+    </section>
 
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+    <!-- Accordion -->
+    <section class="FAQ-section">
+      <div class="FAQ">
+        <h3 class="FAQ-headline">FAQ</h3>
+        <div class="accordion-item-header" id="FAQ1">
+          <h4 class="questions" id="question1">Lobortis feugiat solirum vivamus at augue?</h4>
+        </div>
+        <div class="answers" id="answer1">
+          <p>
+            Sollicitudin ac orci phasellus egestas. Sagittis nisl rhoncus mattis rhoncus urna. Semper feugiat nibh sed pulvinar proin gravida hendrerit lectus. Rutrum quisque non tellus orci ac auctor augue mauris. Facilisis volutpat est velit egestas dui id ornare arcu odio. Neque aliquam vestibulum morbi blandit cursus. Et sollicitudin ac orci phasellus egestas tellus rutrum tellus. Proin nibh nisl condimentum id venenatis a condimentum. Dis parturient montes nascetur ridiculus mus mauris vitae. Enim neque volutpat ac tincidunt. Ultrices dui sapien eget mi proin sed. Elit scelerisque mauris pellentesque pulvinar pellentesque. Urna duis convallis convallis tellus id. Amet luctus venenatis lectus magna fringilla. Etiam non quam lacus suspendisse. Ut faucibus pulvinar elementum integer enim neque volutpat ac. Mauris cursus mattis molestie a iaculis at. Urna et pharetra pharetra massa massa. Sit amet dictum sit amet justo donec. Diam ut venenatis tellus in metus vulputate.
+          </p>
+        </div>
+        <hr>
```

Comment:
Never seen hr tags before so I learned something new! However when I read about them I believe best practice to use instead to get a line is to add a top or bottom border of the accordion-questions to get a line instead. As I understood it you should use it more as a thematic break when several paragraphs have distinct themes and you want to separate them in your code so that someone who reads it can understand that they are about different things.